src/custom-elements.js

```diff
@@ -138,7 +138,7 @@ let Deferred;
       /** @private {!Map<string, !CustomElementDefinition>} **/
       this._definitions = new Map();
 
-      /** @private {!Map<Function, string>} **/
+      /** @private {!Map<Function, !CustomElementDefinition>} **/
```

Comment:
I have changed `this._constructors` to so that it has the entire CustomElementDefinition, since we need more than the tagName now when looking up custom elements by their constructor